src/shared/utils/getDate.ts

```diff
@@ -0,0 +1,25 @@
+function getDate(data: Date) {
```

Comment:
매개변수명은 이런식으로 표현해줘도 좋을 것 같습니다~
```suggestion
function getDate(date: Date) {
```

```suggestion
function getDate(value: Date) {
```